QUARKUS-6233.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-6233 - Allow mongo client to be configured by a tls registry
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6233
+
+Allow mongo client to be configured by a tls registry
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/46293
+
+Documentation:
+- https://quarkus.io/guides/mongodb#tls-configuration
+
+## Scope of testing
+This feature adds a TLS registry configuration for the MongoDB client.
+The tests will verify that the MongoDB client, configured via the TLS registry, functions correctly using mutual TLS (mTLS).
+The testing of TLS is not possible due to mongodb needs certificates from trusted certificate authorities, or it's need to specify the `CAFile` for self-sing certificates.
```

Comment:
Please rephrase. Right now it looks like we will not be testing TLS, which is not correct.